upi/vsphere/vsphere/main.tf

```diff
@@ -0,0 +1,35 @@
+data "vsphere_datacenter" "dc" {
```

Comment:
i personally don't like this `vsphere/vsphere` module. i would prefer it stay in the main

Reply:
@abhinavdahiya ack will fix it.